webapp/channels/src/components/admin_console/admin_settings.tsx

```diff
@@ -240,9 +239,9 @@ export default abstract class AdminSettings <Props extends BaseProps, State exte
                 onSubmit={this.handleSubmit}
             >
                 <div className='wrapper--fixed'>
-                    <AdminHeader>
+                    <div className='admin-console__header'>
```

Comment:
I think it'd actually better to keep `AdminHeader` in case we ever decide to change that component since we can just change the `AdminHeader` instead of finding everything with that class name and updating it. Generally, I like using components for anything used in more than one or two places rather than using raw HTML with a shared class name for that reason.

Reply:
yes makes sense